packages/ui/src/pages/Callback/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+
+import useSocial from '@/hooks/use-social-connector';
+
+type Props = {
+  connector?: string;
+};
+
+const Callback = () => {
+  const { connector } = useParams<Props>();
+  const { socialCallbackHandler } = useSocial();
+
+  useEffect(() => {
+    socialCallbackHandler(connector);
```

Comment:
`window.location.assign` won't work without any user input in android.
Maybe add a button to call `window.location.assign` manually.
Refer to  https://silverhand-io.slack.com/archives/C02A8G4HVAM/p1648893433847489

Reply:
The current pr is just for web redirect only. Will consider this solution on webview